demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -497,6 +495,8 @@ export class DemoMeetingApp
           (document.getElementById('info-name') as HTMLSpanElement).innerText = this.name;
 
           await this.initVoiceFocus();
+          await this.populateAllDeviceLists();
```

Comment:
So it seems we only call `populateAllDeviceLists` once together with `populateVideoFilterInputList`. So do we need to separate `populateVideoFilterInputList` from `populateAllDeviceLists`?

Reply:
The same line of reasoning would suggest combining `initVoiceFocus` with `populateAllDeviceLists`, no? Those are also both called only once and called right here.

I think the names of these functions now accurately describe what they do, each is <50 lines long and is individually understandable, and the point at which they are called accurately represents the order of execution. This is the ideal structure for code.